src/o365/spo/commands/web/web-add.ts

```diff
@@ -0,0 +1,313 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import * as request from 'request-promise-native';
+import GlobalOptions from '../../../../GlobalOptions';
+import { ContextInfo } from '../../spo';
+import {
+  CommandOption,
+  CommandValidate,
+  CommandError,
+  
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Auth from '../../../../Auth';
+import Utils from '../../../../Utils';
+import { PermissionKind, BasePermissions } from './../../common/base-permissions';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  title?: string;
+  webUrl: string;
+  webTemplate?: string;
```

Comment:
webTemplate is not optional, so no need to specify as `webTemplate?: string`. It should be `webTemplate: string` instead